### ISSUE TITLE
2019 edition

### ISSUE THREAD

**Issue by LinguList** [digling/calcblog#3]
It would be useful to start already on the 2019 edition: once these things are entered into a big latex document each time after a post gets published, it is less work (that's my grandmother's rule: rather do things immediately than doing them at once, so you'll never feel there's a lot to do ;).

So maybe @tresoldi, you could set up already a process-2019.py where blog IDs are added.

**Comment by tresoldi** [comment]
Yes, and know that the code is good enough -- there is no way around it, manual editing is needed.

**Comment by LinguList** [comment]
Yes. I know, as I tested the code today. I guess, what the 2019 version should do is: write out a contribution-tex-file per contribution, no complete whole-dump document. And what is done then is: copy-pasting the latex document, and adding those tex-things in form of `\input{}`. This is easier for continuous editing.
